File: RADWIMPS.php
<?php

declare(strict_types=1);

class RADWIMPS
{
    public function then(): self
    {
        echo "前";

        return $this;
    }

    public function 世(): self
    {
        echo "世";

        return $this;
    }
}

(new RADWIMPS())->then()->then()->then()->世();

File: RADWIMPS.swift
class RADWIMPS {
  func then() -> RADWIMPS {
    print("前", terminator: "");
    return self;
  }

  func 世() {
    print("世");
  }
}

let radwimps = RADWIMPS();
radwimps.then().then().then().世();

